File: scramble.js
//scramble letters
function scramble(str, arr){
var newStr = "";
arr.forEach(function(item, index){
	if(str[index]!== item)
		newStr += str[item];
			return newStr;
		});
	return newStr;
}
//scramble("afjdfji",[3,0,6,1,2,5,4]);

File: scramble.rb
#scramble
def scramble(str, arr)
	result = ""
	i = 0
	while i < arr.length
		result += str[arr[i]]
		i += 1
	end
	return result
end

scramble("markov", [5, 3, 1, 4, 2, 0])